dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -676,6 +676,20 @@
 			interrupts = <18 0>;
 			status = "disabled";
 			#io-channel-cells = <1>;
+			has-temp-channel;
+			has-vref-channel;
+		};
+
+		die_temp: dietemp {
+			compatible = "st,stm32-temp-cal";
+			ts-cal1-addr = <0x1FF1E820>;
+			ts-cal2-addr = <0x1FF1E840>;
+			ts-cal1-temp = <30>;
+			ts-cal2-temp = <110>;
+			ts-cal-vrefanalog = <3300>;
+			ts-cal-resolution = <16>;
+			io-channels = <&adc3 18>;
```

Comment:
Unfortunately, channel is not the same for H7 series:
for example H723/H725/H730 is channel 17
Please check channel for all H7 MCU.